guides/micronaut-data-jdbc-repository/micronaut-data-jdbc-repository.adoc

```diff
@@ -1,47 +1,63 @@
 include::{commondir}/common-header.adoc[]
 
-The application exposes some REST endpoints and stores data in a database using Micronaut Data JDBC.
+The application exposes some REST endpoints and stores data in a MySQL database using Micronaut Data JDBC.
 
-include::{commondir}/common-requirements.adoc[]
+== What you will need
```

Comment:
this should be in a common snippet. Please merge `master` against this PR and use the new `common:` syntax.